File: Domain/Photo.cs
using System;

namespace Domain
{
    public class Photo
    {
        public Photo()
        {
            this.Path = "";
            this.Length = 0;
            this.ContentType = "";
        }
        public Guid Id { get; set; }
        public string Path { get; set; }
        public long Length { get; set; }
        public string ContentType { get; set; }
        public virtual AppUser AppUser { get; set; }
    }
}

File: client-app/src/app/models/photo.ts
export interface IPhoto {
  id: string;
  path: string;
  length: number;
  contentType: string;
}

export class Photo implements IPhoto {
  id: string = '';
  path: string = '';
  length: number = 0;
  contentType: string = '';

  constructor(init?: IPhoto) {
    Object.assign(this, init);
  }
}
